src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -120,52 +138,65 @@ const styles = StyleSheet.create({
     numberOfLabelsColumnWidth: {
         width: "20%",
     },
+    collectionOrDeliveryHeader: {
+        marginRight: "5px",
+        marginBottom: "5px",
+        alignSelf: "flex-start",
+    },
 });
 
 const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ data }) => {
-    const header = (
-        <View style={[styles.tableHeader, styles.flexRow]}>
-            <View style={[styles.tableColumn, styles.nameColumnWidth]}>
-                <Text>Name</Text>
-            </View>
-            <View style={[styles.tableColumn, styles.addressColumnWidth]}>
-                <Text>Address</Text>
-            </View>
-            <View style={[styles.tableColumn, styles.contactColumnWidth]}>
-                <Text>Contact</Text>
-            </View>
-            <View style={[styles.tableColumn, styles.packingDateColumnWidth]}>
-                <Text>Packing Date</Text>
-            </View>
-            <View style={[styles.tableColumn, styles.instructionsColumnWidth]}>
-                <Text>Instructions</Text>
-            </View>
-            <View style={[styles.tableColumn, styles.numberOfLabelsColumnWidth]}>
-                <Text>Number of Parcels</Text>
+    const createHeader = (category: string): React.JSX.Element => {
+        return (
+            <View style={[styles.tableHeader, styles.flexRow]}>
+                <View style={[styles.tableColumn, styles.nameColumnWidth]}>
+                    <Text>Name</Text>
+                </View>
+                <View style={[styles.tableColumn, styles.addressColumnWidth]}>
+                    <Text>{category === "Collection" ? "Collection Centre" : "Address"}</Text>
```

Comment:
```suggestion
    const createHeader = (category: Method): React.JSX.Element => {
        return (
            <View style={[styles.tableHeader, styles.flexRow]}>
                <View style={[styles.tableColumn, styles.nameColumnWidth]}>
                    <Text>Name</Text>
                </View>
                <View style={[styles.tableColumn, styles.addressColumnWidth]}>
                    <Text>{category === Method.Collection ? "Collection Centre" : "Address"}</Text>
```

Reply:
Since you mentioned you haven't used enums before - it's best practice to never look at the actual values of them, i.e. all the comparison etc is done through the enum itself. This is good because the compiler can catch any problems like you using a value that shouldn't be available or having a typo and then spending ages trying to find the issue. It also means you can update the enum value in one place and it will change everywhere rather than having to update everything manually. Very powerful tool!